test/ng/directive/inputSpec.js

```diff
@@ -931,6 +946,22 @@ describe('input', function() {
     });
 
 
+    it('should be possible to override the timezone', function() {
+      var inputElm = helper.compileInput('<input type="week" ng-model="value" ng-model-options="{timezone: \'UTC\'}" />');
+
+      helper.changeInputValueTo('2013-W03');
+      expect(+$rootScope.value).toBe(Date.UTC(2013, 0, 17));
+
+      inputElm.controller('ngModel').$overrideModelOptions({timezone: '+5000'});
+
+      $rootScope.$apply(function() {
+        // the 17. with an offset of +5000 moves the date into next week
```

Comment:
Can you expand a bit on this comment? It mentions `17.` (which is in German format 😛), but the code uses `18`. Also, how does it move to the next week?